### PR TITLE
Enable op Grouping in runtime

### PR DESCRIPTION
**Pull request by vladsud**
Op grouping has been proven in production by being enabled at 100% in PROD.
It's time to remove feature gate / flights and make in On by default.

This has similar implications as compression on AzureClient 1.3 compatibility. These options can't be abled for clients who have AzureClient 1.3 in prod. This needs to be resolved in some way before 2.0 ships. Such solution will apply to all similar 2.0 configs, in other words enabling op grouping by default does not make this problem worse, and solution has to be put in place even if op grouping was not enabled by default.

Please see https://github.com/microsoft/FluidFramework/pull/20283 for more discussions / possible direction.

There is some due diligence required before it can be merged.
I need to better understand if feature gates were impacting Loop only, or all of our 1st party customers (WB included).
In other words, if there is any unique impact for WB.

### CODE REVIEW

**Comment by vladsud** [comment]
> The grouped batching feature requires that 99.9% of runtime versions of the application are at least "2.0.0-internal.7.0.3". Are we pushing that responsibility to the app developer to check this and subsequently disable the feature if necessary?

That was next on my agenda to figure out.
No, if we do not have saturation, we should not enable it. At the same time all reports I saw said that we are already at 100% deployment of that feature for Loop, which would mean we already reached that level of saturation.

Here is the data for !WB. Something does not add up in this story. Either min version you mentioned, or that we enabled op grouping, or that we did it the right way. Still looking.

<html>
<body>
<span>Execute in [<a href="https://dataexplorer.azure.com/clusters/kusto.aria.microsoft.com/databases/6a8929bcfc6d44e9b13fee392ada9cf0?query=H4sIAAAAAAAAA62SS0%2FDMBCE7%2F0VS05O1YakB8qlXHgcAaEKjtHW3jaWGhutN30gfjxuA7SHSIiqPliWd%2BYbWePGWe%2FgaT63msqHZWNNu780TmxNZb%2F3CeuKmOB%2BRU7KabyEG8CFV2OT%2Fg5tcF6ofpetukPBUjOh0K13gtYRf9NeiUOMO9j22soHecSIvZhA8lZZoZlHNklU0UbIGVi1PpiAxkCqB3H9HQNBkCWsrVSQjLI8y4fWCbHDZXadDLruzkAed5DHZyF3gPPh1WnoYk8rss1p9nxvz%2F9h33UemrpGth8E2sehSmG2%2Fek2jjHA9ND5M7GOHw4XFGvnqDeqyOPjod%2B6S7gE8UHjEllN4ZgeT6rVpJAOYLTL9myIj%2FLAUNBfDGLi3v0CAAA%3D">Web</a>] 
[<a href="https://kusto.aria.microsoft.com/6a8929bcfc6d44e9b13fee392ada9cf0?query=H4sIAAAAAAAAA62SS0%2FDMBCE7%2F0VS05O1YakB8qlXHgcAaEKjtHW3jaWGhutN30gfjxuA7SHSIiqPliWd%2BYbWePGWe%2FgaT63msqHZWNNu780TmxNZb%2F3CeuKmOB%2BRU7KabyEG8CFV2OT%2Fg5tcF6ofpetukPBUjOh0K13gtYRf9NeiUOMO9j22soHecSIvZhA8lZZoZlHNklU0UbIGVi1PpiAxkCqB3H9HQNBkCWsrVSQjLI8y4fWCbHDZXadDLruzkAed5DHZyF3gPPh1WnoYk8rss1p9nxvz%2F9h33UemrpGth8E2sehSmG2%2Fek2jjHA9ND5M7GOHw4XFGvnqDeqyOPjod%2B6S7gE8UHjEllN4ZgeT6rVpJAOYLTL9myIj%2FLAUNBfDGLi3v0CAAA%3D&web=0">Desktop</a>]</span>
</div>
</body>
</html>

version | count_ | Percentage
-- | -- | --
2.0.0-rc.1.0.8 | 180,468 | 0.37
2.0.0-internal.8 | 35,942,650 | 73.35
2.0.0-internal.7 | 10,204,190 | 20.82
2.0.0-internal.0-6 | 2,468,101 | 5.04
1.x | 150,451 | 0.31
0.x | 56,538 | 0.12

File: .changeset/slick-symbols-dress.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+Make op grouping On by default
+
+Op grouping feature reduces number of ops on the wire by grouping all ops in a batch. This allows applications to substantially reduce chances of being throttled by service when sending a lot of ops.
+This feature could be enabled only by applications that have consumed 2.0.0-internal.7.0.2 version and have application version based on it saturated in the marker (to 99.99% or higher). Enabling it too soon will result on old client crashing when processing grouped ops.
+
+The feature has been proven in production in Loop app, as it was enabled through feature gates at 100% in PROD.
+All internal applications (Loop, Whiteboard) that send telemetry to our common Kusto tenant are already at or above minimal required version of runtime.
+
+If your application does not satisfy these deployment requirements, please disable op grouping via passing IContainerRuntimeOptions.enableGroupedBatching = false when calling ContainerRunntime.load().
```
Comment (kian-thompson) [inline .changeset/slick-symbols-dress.md:13]:
```suggestion
If your application does not satisfy these deployment requirements, please disable op grouping via passing IContainerRuntimeOptions.enableGroupedBatching = false when calling ContainerRuntime.load().
```

**Review by kian-thompson — APPROVED** [review]

**Comment by alexvy86** [comment]
This seems to have fixed our e2e perf tests against local server, where some tests were seemingly getting stuck with error `Runtime detected too many reconnects with no progress syncing local ops.` ([example](https://dev.azure.com/fluidframework/internal/_build/results?buildId=262339&view=logs&j=d042b387-7692-5f02-a062-ee417cc37fe5&t=c9404925-d8e2-5bd7-253e-b1cab12afc6d&l=548); msft internal). 

image

Does that seem like an expected side effect? Not complaining :) just trying to understand why those tests now complete successfully.
